make model docs device agnostic (2) (#40256)
huggingface/transformers: docs/source/en/quicktour.md

</hfoption>
<hfoption id="image segmentation">

> Use `infer_device` to automatically detect available hardware(e.g. CUDA GPU, Intel XPU) and use it to accelerate inference.

Reviewer: ```suggestion
Use [`~infer_device`] to automatically detect an available accelerator for inference.
```

</hfoption>
<hfoption id="automatic speech recognition">

> Use `infer_device` to automatically detect available hardware(e.g. CUDA GPU, Intel XPU) and use it to accelerate inference.

Reviewer: ```suggestion
Use [`~infer_device`] to automatically detect an available accelerator for inference.
```
